crates/common/src/config.rs

```diff
@@ -125,6 +127,20 @@ pub enum NetworkConfig {
     },
 }
 
+/// Configuration for the [Constraints API](https://docs.boltprotocol.xyz/technical-docs/api/builder)
+///
+/// These checks are enabled by default. Disabling them is recommended only for testing purposes.
+#[derive(Serialize, Deserialize, Clone)]
+pub struct ConstraintsApiConfig {
+    pub check_constraints_signature: bool,
```

Comment:
can you add a doc comment explaining that this removes the sig verification check on constraints? 
It's almost obvious but better to be clear.

Reply:
Done, lmk if it's okay!

Reply:
Yesss